scripts/deploy.js

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env node
+
+const fs = require('fs')
+const childProcess = require('child_process')
+
+const PACKAGES = ['@contentful/create-contentful-app', 'create-contentful-app', '@contentful/cra-template-create-contentful-app']
+const ORIGINAL_PACKAGE_JSON = require('../package.json')
+
+if (!process.env.NPM_TOKEN) {
+  throw new Error('Missing NPM_TOKEN!')
+}
+
+for (const package of PACKAGES) {
+  console.log('')
+  console.log('📦 Deploying package:', package)
+  const packageJson = {...ORIGINAL_PACKAGE_JSON, name: package}
+
+  console.log(` > 📝 Updating package.json with name: ${package}...`)
+  fs.writeFileSync('../package.json', JSON.stringify(packageJson))
+
+  console.log(` > ⚙️  Updating package-lock.json by means of "npm i"...`)
+  childProcess.execSync('npm i', {cwd: '../'})
+
+  console.log(` > 📚 Publishing on the registry...`)
+  childProcess.execSync('npm publish', {cwd: '../'})
```

Comment:
Is it piping things like 2-factor auth code entry back to the original shell?

Reply:
Uhm, I need to double check actually. I think I have inserted my code soon before, so it didn't ask and went through